com.termius.termius.appdata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- © 2019 Termius Corporation.> -->
+<component>
+ <id>com.termius.termius.desktop</id>
```

Comment:
This is not the recommended format for an app-id check through https://github.com/flathub/flathub/wiki/AppData-Guidelines

Reply:
Oh thanks for information have fixed that